python/pyspark/pandas/frame.py

```diff
@@ -882,6 +882,9 @@ def add(self, other) -> "DataFrame":
     spark = CachedAccessor("spark", SparkFrameMethods)
 
     # create accessor for pandas-on-Spark specific methods.
+    pandas_on_spark = CachedAccessor("pandas_on_spark", PandasOnSparkFrameMethods)
+
+    # keep the name "koalas" for backward compatibility.
     koalas = CachedAccessor("koalas", PandasOnSparkFrameMethods)
```

Comment:
Can we deprecate this and show a deprecated warning when users call this?

Reply:
Sure, I'll add it

Reply:
@HyukjinKwon , I just created JIRA to document this deprecation: SPARK-35569.

I think we can do this after completing the documentation porting.